tagbot/action/changelog.py

```diff
@@ -103,10 +106,15 @@ def _pulls(self, start: datetime, end: datetime) -> List[PullRequest]:
             p for p in self._issues_and_pulls(start, end) if isinstance(p, PullRequest)
         ]
 
-    def _custom_release_notes(self, version: str) -> Optional[str]:
+    def _custom_release_notes(self, version_tag: str) -> Optional[str]:
         """Look up a version's custom release notes."""
         logger.debug("Looking up custom release notes")
-        pr = self._repo._registry_pr(version)
+
+        tag_prefix = self._repo._tag_prefix()
+        i_start = len(tag_prefix) - 1
+        package_version = version_tag[i_start:]
+
+        pr = self._repo._registry_pr(package_version)
```

Comment:
```suggestion
        tag_prefix = self._repo._tag_prefix()
        i_start = len(tag_prefix) - 1
        package_version = version_tag[i_start:]
        pr = self._repo._registry_pr(package_version)
```

nitpick but i tend to not like blank lines in our precious significant-whitespace python, it makes it a bit harder to see the levels of indentation